sh tools/substitutions.sh (#1040)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc

`publisher`: Name of the publisher.
`repository`: URL of the source repository.
`title`: Title (long).
`type`: `{prod-short} Plugin`, `VS Code extension`.
`version`: Version (example\: `7.5.1`).
`spec`: Specifications (see underneath).
:===

.`spec` attributes
:===
`endpoints`: plugin endpoints

> `containers`: sidecar containers for plugin. `{prod-short} Plugin` and `VS Code extension` supports only one container.

Reviewer: ```suggestion
`containers`: sidecar containers for plug-in. `{prod-short} plug-in` and `VS Code extension` supports only one container.
```

Author: I have to revert `Che Plugin` and `Che editor` types.

Author: This is confusing: type for a `{prod-short} plug-in` can be: `Che Plugin`, `Che Editor`, `VS Code extension`. Depending on the context we may or may not use the attribute. All my attempts to automate the substitution with regex failed so far. I need to reassess everything manually each time :/.

:===

.Example `meta.yaml` for a Che-Theia plug-in: the {prod-short} machine-exec Service
[source,yaml]
----
apiVersion: v2
category: Other

> description: Che Plugin with che-machine-exec service to provide creation terminal or tasks for {prod} workspace containers.

Reviewer: This sentence is giving me a headache :(

Author: It is a verbatim copy of a real metadata, I am sorry :/. Should we review them?

:===

.Example `meta.yaml` for a Che-Theia plug-in: the {prod-short} machine-exec Service
[source,yaml]
----
apiVersion: v2
category: Other

> description: {prod-short} Plug-in with che-machine-exec service to provide creation terminal or tasks for Eclipse CHE workspace containers.

Reviewer: ```suggestion
description: {prod-short} plug-in with che-machine-exec service to provide creation terminal or tasks for {prod} workspace containers.
```

displayName: {prod-short} machine-exec Service
firstPublicationDate: "2019-12-02"
icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
name: che-machine-exec-plugin
publisher: eclipse
repository: https://github.com/eclipse/che-machine-exec/
title: Che machine-exec Service Plugin

> type: {prod-short} Plugin

Reviewer: ```suggestion
type: {prod-short} plug-in
```

Reviewer: Hmmm... looking at this again... is `Che Plugin` a defined 'type'? If it is, then neither an attribute for prod name, nor the 'plug-in' form should be used.

Author: ```suggestion
type: Che Plugin
```

Author: `type: Che Plugin` indeed see this example: https://github.com/eclipse/che-plugin-registry/blob/master/v3/plugins/eclipse/che-machine-exec-plugin/7.7.0/meta.yaml
